### PR TITLE
Bug#1493091 - Add addition information about F5 router

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1493091

- Highlighted not supported section and added more points under it

### CODE REVIEW

**Comment by lihongan** [comment]
@rajatchopra @knobunc  please also help to take a look.

**Comment by gaurav-nelson** [comment]
@rajatchopra @knobunc bump

File: install_config/router/f5_router.adoc [hunk 1]
```diff
@@ -65,11 +65,19 @@ endif::[]
...
+====
+The following features are not supported with *F5 BIG-IP®*:
+
+* Wildcard routes together with re-encrypt - you must supply a certificate and key in the route. And if you provide a certificate, a key and a CA, the CA is never used.
+* A pool is created for all services, even for the ones with no associated route.
 * xref:../../admin_guide/idling_applications.adoc#admin-guide-idling-applications[Idling applications]
-* Re-ecrypt routes
-* Unencrypted HTTP traffic in allow and redirect mode, with edge TLS termination
-* Certificates specifically configured for particular routes
+* Re-encrypt routes.
```
Comment (rajatchopra) [inline install_config/router/f5_router.adoc]: Re-encrypt routes are supported.

**Review by rajatchopra — CHANGES_REQUESTED** [review]

File: install_config/router/f5_router.adoc [hunk 2]
```diff
@@ -65,11 +65,19 @@ endif::[]
...
+* Unencrypted HTTP traffic in _allow_ and _redirect_ mode, with edge TLS termination. (`insecureEdgeTerminationPolicy (Allow/Redirect)`)
+* Certificates specifically configured for particular routes.
```
Comment (rajatchopra) [inline install_config/router/f5_router.adoc]: Not sure what this means. We do take specific certs for edge/re-encrypt routes.

**Comment by gaurav-nelson** [comment]
@rajatchopra @knobunc 
It would be great if I can get list of things that are supported and not supported.

**Comment by rajatchopra** [comment]
This PR looks good. Coming  up with a comprehensive comparison will take dedicated effort. Get this merged, I say because this information is correct. Eventually we want container-connector from F5 to take over anyway.

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL

**Comment by gaurav-nelson** [comment]
minor changes no revision history needed

File: install_config/router/f5_router.adoc [hunk 3]
```diff
@@ -65,12 +65,17 @@ endif::[]
 * 11.x
 * 12.x
 
-However, the following features are not supported with *F5 BIG-IP®*:
+[IMPORTANT]
+====
+The following features are not supported with *F5 BIG-IP®*:
 
+* Wildcard routes together with re-encrypt - you must supply a certificate and key in the route. And if you provide a certificate, a key and a CA, the CA is never used.
```
Comment (mburke5678) [inline install_config/router/f5_router.adoc]: Should this be _Wildcard routes together with re-encryption_? Seems like something is missing here. The [3.6 release notes](https://github.com/openshift/openshift-docs/blob/f6a965f6af48b852f838f12160ca42aff3ffb5c7/release_notes/ocp_3_6_release_notes.adoc) talk about _re-encrypt routes_.

Comment (mburke5678) [reply install_config/router/f5_router.adoc]: s/a certificate, a key and a CA/a certificate, a key, and a CA ??

**Comment by mburke5678** [comment]
@gaurav-nelson A few nits. LGTM

**Comment by gaurav-nelson** [comment]
Thanks @mburke5678 

minor updates no rev history needed.

**Pull request merged**